packages/table/src/ExpandedContent/ExpandedContent.styles.ts

```diff
@@ -0,0 +1,43 @@
+import { TransitionStatus } from 'react-transition-group';
+
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+
+export const baseStyles = css`
+  padding: 0;
+  overflow: hidden;
+`;
+
+export const expandedContentStyles: Record<Theme, string> = {
+  [Theme.Dark]: css`
+    background-color: ${palette.gray.dark4};
+  `,
+  [Theme.Light]: css`
+    background-color: ${palette.gray.light3};
+  `,
+};
+
+const _hiddenStyles = css`
```

Comment:
why `_`

Reply:
this is Adam's touch, but it made sense to me since this is an internal style utility that doesn't even get exported